lib/matplotlib/figure.py

```diff
@@ -1347,6 +1347,31 @@ def draw_artist(self, a):
     def get_axes(self):
         return self.axes
 
+    def inaxes(self, xy):
+        """
+        Check if a point is in an axes.
+
+        Parameters
+        ----------
+        xy : tuple or list
+            (x,y) coordinates.
+            x position - pixels from left of canvas.
+            y position - pixels from bottom of canvas.
+
+        Returns
+        -------
+        axes: topmost axes containing the point, or None if no axes.
+
+        """
+        axes_list = [a for a in self.get_axes() if a.patch.contains_point(xy)]
+
+        if axes_list:
+            axes = max(reversed(axes_list), key=lambda x: x.zorder)
```

Comment:
For consistency we probably should use `_topmost_artist` here (so if we ever need to change that logic we can do it exactly once).